tensorflow/core/util/use_cudnn.cc

```diff
@@ -90,4 +90,13 @@ FP16ConvMode CudnnConvComputeMode() {
   return FP16ConvMode::kAccurate;
 }
 
+// Use CuDNN depthwise conv when filter is 1x1, 3x3, 5x5, or 7x7.
+// See cudnn release note 7.6.3. (https://docs.nvidia.com/deeplearning/sdk/c
+// udnn-release-notes/rel_763.html#rel_763)
```

Comment:
The comment should be on the function decl (in the header) and be descriptive "Returns true if ...".

Reply:
TensorFlow follows the Google C++ style: https://google.github.io/styleguide/cppguide.html#Function_Comments